src/content/docs/custom-pipeline/setup.mdx

```diff
@@ -1,209 +1,186 @@
 ---
 title: Setting up your computer
-description: Setting up your computer for creating a custom pipeline.
+description: Setting up your computer to create a custom pipeline.
 ---
 
-## Prototype pipeline development container
+import CloneIcon from '~icons/codicon/source-control'
+import RemoteIcon from '~icons/codicon/remote'
+import ExtensionsIcon from '~icons/codicon/extensions'
+import CoffeeIcon from '~icons/codicon/coffee'
+import WaitIcon from '~icons/ic/round-access-time'
 
-## Requirements
+## Quick setup : VSCode and **devcontainers**
 
-- [VS Code](https://code.visualstudio.com) &geq; 1.95
-- [Docker](https://www.docker.com/get-started/) &geq; 24 (we recommend using [Docker Desktop](https://www.docker.com/products/docker-desktop))
+### Prerequisites
 
-### Configure Docker access
-
-The simplest way of installing everything Docker is to use [Docker Desktop](https://www.docker.com/products/docker-desktop).
-You can also go the [engine way](https://docs.docker.com/engine/install) and install Docker manually.
-
-Once installed, you need to configure some access rights to the Docker daemon. The easiest way to do this is to add your user
-to the `docker` group. This can be done with the following command :
+The fastest way to start coding is by installing [VSCode](https://code.visualstudio.com) (&geq; 1.95)
+and [Docker](https://www.docker.com/get-started/) (&geq; 24). Once done, open **VSCode**. If it opens
+in a project, close it or open a **new empty window** (`ctrl+shift+N`). Then, test that you have access
+to **docker**. In the `VSCode Terminal`, run :
 
 ```bash
-sudo groupadd docker
-sudo usermod -aG docker $USER
+docker run hello-world
 ```
 
-After running this command, you need to log out and log back in your terminal to apply the changes.
-
-## Usage
+![docker run hello-world](../../../assets/images/setup-test-docker.gif)
 
-First, create a `.devcontainer` directory at your project's root.
-```bash
-mkdir <project_dir>/.devcontainer
-```
-
-:::caution
-The `prototyping` environment definition is not meant to be run from the `nf-neuro` repository root. If you downloaded or
-cloned the `nf-neuro` repository, don't place your project at its root (we recommend putting your project outside its
-directory to prevent conflicts with `git`).
+:::tip[Can't connect to the daemon socket]
+If you get an error, either the **docker daemon** is not running or you need to
+setup some permissions to access it. Follow [these instructions](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user),
+then get back here !
 :::
 
-### Get the devcontainer configuration
-
-If you downloaded or cloned `nf-neuro` locally :
-
-- Copy the `prototyping` directory and its content, located in `<nf-neuro root>/.devcontainer`, to the `.devcontainer`
-  directory in your project :
-  ```bash
-  cp -r <nf-neuro root>/.devcontainer/prototyping <project_dir>/.devcontainer/.
-  ```
+### Custom pipeline template
```

Comment:
I would change it to Clone or Get pipeline template

Reply:
How about **Install the pipeline tempalte** ?